src/sidebar/components/Annotation/test/AnnotationEditor-test.js

```diff
@@ -36,6 +39,17 @@ describe('AnnotationEditor', () => {
     fakeAnnotationsService = {
       save: sinon.stub(),
     };
+
+    fakeGroup = {
+      name: 'Fake Group',
+      type: 'private',
+    };
+
+    fakeMetadata = {
+      isReply: sinon.stub().returns(false),
+      isSaved: sinon.stub().returns(true),
+    };
```

Comment:
I would be inclined to use "real" replies and saved annotations rather than mocking these very simple utilities. `isSaved` just checks whether the annotation has a non-empty ID and `isReply` checks for a non-empty `references` list.